lesson3/Homework-Slava/main 2.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <cmath>
+
+
+double y(double x) {
+    if (x >= 5) {
+        return 5 * sqrt(x) + 8.2;
```

Comment:
For clean code there should not be more than one return in a function, please fix it by creating one variable and then returning it. Everything is perfect